README.md

```diff
@@ -197,6 +197,12 @@ Everything will immediately show up in ImHex's Content Store and gets bundled wi
 | ZIP  | `application/zip` | [`patterns/zip.hexpat`](patterns/zip.hexpat) | End of Central Directory Header, Central Directory File Headers |
 | ZLIB | `application/zlib` | [`patterns/zlib.hexpat`](patterns/zlib.hexpat) | ZLIB compressed data format |
 | ZSTD | `application/zstd` | [`patterns/zstd.hexpat`](patterns/zstd.hexpat) | Zstandard compressed data format |
+| MOD | `3d-model/mod` | [`patterns/DMC3HD-Mod.hexpat`](patterns/dmc3_hd_mod.hexpat) | 3D Model files used in Devil May Cry 3 HD Collection |
```

Comment:
These should be alphabetically ordered. Also the name you specified here is not really standard, I'd name them for what they are. For example "Devil May Cry 3 3D Model" or `Assassin's Creed Unity Forge Archive`

Reply:
Or if you move them to a subfolder, simply name it "Assassin's Creed: Unity"